package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "bfx-hf-strategy-dazaar",
+  "version": "1.0.0",
+  "description": "Dazaar logic for bfx-hf-strategy",
+  "main": "./index.js",
+  "author": "Bitfinex",
+  "contributors": [
+    "Cris Mihalache <cris.m@bitfinex.com> (https://www.bitfinex.com)",
+    "Paolo Ardoino <paolo@bitfinex.com> (https://www.bitfinex.com)",
+    "Jacob Plaster <jacob.plaster@bitfinex.com> (https://www.bitfinex.com)",
+    "Robert Kowalski <robert@bitfinex.com> (https://www.bitfinex.com)"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "lint": "standard",
+    "test": "npm run lint && npm run unit",
+    "unit": "NODE_ENV=test mocha -R spec -b --recursive",
+    "api_docs": "node_modules/jsdoc-to-markdown/bin/cli.js ./index.js > docs/api.md",
+    "docs": "npm run api_docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bitfinexcom/bfx-hf-strategy-dazaar.git"
+  },
+  "keywords": [
+    "honey framework",
+    "bitfinex",
+    "bitcoin",
+    "BTC",
+    "dazaar"
+  ],
+  "dependencies": {
+    "bfx-api-node-models": "^1.2.11",
+    "bfx-api-node-util": "^1.0.9",
+    "bfx-hf-strategy": "^1.0.5",
+    "debug": "^4.1.1"
+  },
+  "devDependencies": {
+    "@dazaar/payment": "^1.0.2",
```

Comment:
A lot of these shouldn't be in devDeps right?

Reply:
they are just used for the example-code, for the main lib just debug, bfx-hf-strategy, bfx-api-node-models, bfx-api-node-util are required

(in the files index.js and util.js)